flow/envs/multiagent/__init__.py

```diff
@@ -10,7 +10,8 @@
 from flow.envs.multiagent.traffic_light_grid import MultiTrafficLightGridPOEnv
 from flow.envs.multiagent.highway import MultiAgentHighwayPOEnv
 from flow.envs.multiagent.merge import MultiAgentMergePOEnv
-from flow.envs.multiagent.i210 import I210MultiEnv, MultiStraightRoad
+from flow.envs.multiagent.i210 import I210MultiEnv, MultiStraightRoad, I210MADDPGMultiEnv, MultiStraightRoadMADDPG
```

Comment:
I210MADDPGMultiEnv import doesn't work. Try running `python train.py multiagent_i210 test` to verify